lib/explorer/data_frame.ex

```diff
@@ -2973,6 +2973,112 @@ defmodule Explorer.DataFrame do
     Shared.apply_impl(df, :drop_nil, [columns])
   end
 
+  @doc """
+  Relocates columns.
+
+  Change column order within a DataFrame. The `before` and `after` options are mutually exclusive.
+  Providing no options will relocate the columns to beginning of the DataFrame.
+
+  ## Options
+
+    * `:before` - Specifies to relocate before the given column.
+
+    * `:after` - Specifies to relocate after the given column.
```

Comment:
```suggestion
    * `:before` - Specifies to relocate before the given column.
      It can be an index or a column name.

    * `:after` - Specifies to relocate after the given column.
      It can be an index or a column name.
```